x-pack/build_chromium/linux/args.gn

```diff
@@ -4,3 +4,5 @@ symbol_level = 0
 is_component_build = false
 remove_webcore_debug_symbols = true
 enable_nacl = false
+# Please, consult @elastic/kibana-security before changing/removing this option.
+use_kerberos = false
```

Comment:
__note:__ we don't need to rebuild/republish binary for linux even though I added this option here, since current binary inherited `use_kerberos = false` via `import("//build/args/headless.gn")`. I'm adding it here for clarity and to not depend on default values defined somewhere else.